acceptance/bundle/resources/clusters/deploy/update-and-resize-autoscale/script

```diff
@@ -0,0 +1,58 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+    rm out.requests.txt
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
```

Comment:
Could we also add coverage for "bundle plan" and "bundle debug plan > out.plan_<op>.$DATABRICKS_BUNDLE_ENGINE.json"?